pkg/controller/config/v1alpha1/doc.go

```diff
@@ -0,0 +1,6 @@
+// +k8s:deepcopy-gen=package
+// +k8s:defaulter-gen=TypeMeta
+
+// Package v1alpha1 contains the API for configuring the os-coreos extension.
```

Comment:
This should probably also include a go:generate statement to create docs for the CRDs like for example in gardener-extension-provider-openstack.